openquake/hazardlib/shakemap/parsers.py

```diff
@@ -636,7 +636,8 @@ def _contents_properties_shakemap(usgs_id, user, monitor):
     shakemap = _get_preferred_shakemap(properties['products']['shakemap'])
     contents = shakemap['contents']
 
```

Comment:
I guess we can replace `getattr(user, 'level', 0)` with `user.level` now.

Reply:
Done